concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3599,6 +3613,10 @@ jobs:
           - get: cf-manifest
             passed: ['post-deploy']
 
+            # FIXME: when there is a release that supports CF CLI v7 we can use
+            # the submodule in this repo again
+          - get: app-autoscaler-release-git
```

Comment:
I'm a bit confused how things work by just `get`ting this. Is undoing it simply a matter of not doing the `get` or am I missing something?

Great work either way

Reply:
Oh, https://github.com/alphagov/paas-cf/pull/2444/commits/7b62da657a413204f8aff375dbdb4e77c750fcb3#diff-7f47ee629d360efc30a60c7fc2affb52R59 explained